com.unity.render-pipelines.high-definition/Runtime/Core/CoreResources/BlitCubeTextureFace.shader

```diff
@@ -59,7 +60,9 @@ Shader "Hidden/SRP/BlitCubeTextureFace"
 
             float4 frag (Varyings input) : SV_Target
             {
-                return SAMPLE_TEXTURECUBE_LOD(_InputTex, sampler_InputTex, input.texcoord, _LoD);
+                float4 color = SAMPLE_TEXTURECUBE_LOD(_InputTex, sampler_InputTex, input.texcoord, _LoD);
+                color.rgb = DecodeHDR(color, _InputTex_HDR);
```

Comment:
See my question below but who control _InputTex_HDR ? 
In case we blit a real time cubemap we will not used the compression of the quality settings but the current format (fp16 or 111110). We need to be sure that when using those format we aren't affected.

QA should do a pass on this. 

The _InputTex_HDR should currently be setup by the ConvertTexture call below based on the required format on SRP side.